app/screens/wallet/Vault.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import { shell } from 'electron';
+import React, { Component } from 'react';
+import { NewVault, VaultType } from '/components/wallet';
+import { smColors } from '/vars';
+import styled from 'styled-components';
+import Link from '../../basicComponents/Link';
+import Button from '../../basicComponents/Button';
+
+const isDarkModeOn = localStorage.getItem('dmMode') === 'true';
+
+const Wrapper = styled.div`
```

Comment:
instead Wrapper use CorneredContainer from 'pos-ui' branch, it has properties for header and sub header text and icon, change prop types if required (component and not just plain text and etc)

Reply:
you can see usage of this component in NodeSetup.js in same branch

Reply:
But our vault is part of wallet screen(i thought), so in this case we will have double CorneredContainer - one in wallet component, and other in the Vault Component

Reply:
CorneredContainer is in 'components/common' folder so it can be used anywhere in components. Checkout branch 'pos-ui' as it has the latest ui components version and use it as base for this and future prs to save the overhead of creating existing components

Reply:
Also run 'npm run lint' to prevent pr failure in Travis because of lint issues

Reply:
fix